src/components/manager/TableTreeSelect.js

```diff
@@ -2,10 +2,11 @@ import React, {Component} from "react";
 import {connect} from "react-redux";
 import PropTypes from "prop-types";
 
-import {Spin, Tag, TreeSelect} from "antd";
+import {Spin, TreeSelect} from "antd";
 
 import {nop} from "../../utils/misc";
 
+// TODO: refactor to dataset select or remove?
```

Comment:
i did some work on this in my web branch which probably merge conflicts now, but maybe worth looking at?

Reply:
Great, will look into it!